### PR TITLE
create www-group if doesnt exists group

### PR DESCRIPTION
**Pull request by pschoffer**
The `php-fpm` container actually needs the `www-data` group too.

This change makes sure group exists before creating user and creates it otherwise.

### CODE REVIEW

File: dev-tools/scripts/user-container-setup.sh [hunk 1]
```diff
@@ -35,10 +36,26 @@ if [ "$LANDO_WEBROOT_UID" != "$LANDO_HOST_UID" ]; then
     fi
 
     userdel "$LANDO_WEBROOT_USER" > /dev/null 2>&1
```
Comment (pschoffer) [inline dev-tools/scripts/user-container-setup.sh:38]: `userdel` would delete group associated with the user if this was the only user (which it is on php-fpm). So we need to create the group only after this line.

File: dev-tools/scripts/user-container-setup.sh [hunk 2]
```diff
@@ -35,10 +36,26 @@ if [ "$LANDO_WEBROOT_UID" != "$LANDO_HOST_UID" ]; then
...
-    useradd --uid "$LANDO_HOST_UID" -M -N "$LANDO_WEBROOT_USER"
+
+    lando_info "Making sure group $LANDO_WEBROOT_GROUP exists"
+
+    cat /etc/group | grep -q "$LANDO_WEBROOT_GROUP"
+    if [ $? != 0 ]; then
+        lando_warn "Group $LANDO_WEBROOT_GROUP doesn't exist. Will attmpt to creat it."
```
Comment (pauarge) [inline dev-tools/scripts/user-container-setup.sh:44]:
```suggestion
        lando_warn "Group $LANDO_WEBROOT_GROUP doesn't exist. Will attempt to create it."
```

File: dev-tools/scripts/user-container-setup.sh [hunk 3]
```diff
@@ -35,10 +36,26 @@ if [ "$LANDO_WEBROOT_UID" != "$LANDO_HOST_UID" ]; then
...
+            exit 1;
+        fi
+    fi
+
+    useradd --uid "$LANDO_HOST_UID" -M -N -G "$LANDO_WEBROOT_GROUP" "$LANDO_WEBROOT_USER"
     if [ $? = 0 ]; then
         lando_info "SUCCESS: user was added"
     else
         lando_error "User was not added"
+        exit 2;
```
Comment (pauarge) [inline dev-tools/scripts/user-container-setup.sh:59]: Why are we doing `exit 2` here and `exit 1` above?

Comment (pschoffer) [reply dev-tools/scripts/user-container-setup.sh:59]: Just to differentiate the fails? Are there actually any good practices around shell returned values? I am fine with haveing 1 on both if you think that is better.

Comment (pauarge) [reply dev-tools/scripts/user-container-setup.sh:59]: [The return codes have meaning](https://tldp.org/LDP/abs/html/exitcodes.html). I _think_ in both cases we are fine with `1`.

Comment (pschoffer) [reply dev-tools/scripts/user-container-setup.sh]: Oh I see. Thank you! fixed

**Review by pauarge — APPROVED** [review]

**Pull request merged**